openshift-storage-libs/openshiftstoragelibs/openshift_ops.py

```diff
@@ -352,14 +352,11 @@ def oc_create_pvc(hostname, sc_name=None, pvc_name_prefix="autotests-pvc",
 
 
 def _oc_create_app_dc_with_io_image(hostname, pvc_name, dc_name_prefix,
-                                    replicas, space_to_use, label, image):
+                                    replicas, space_to_use, label, image,
+                                    cmd_args=None):
     dc_name = "%s-%s" % (dc_name_prefix, utils.get_random_str())
-    container_data = {
-        "name": dc_name,
-        "image": image,
-        "volumeMounts": [{"mountPath": "/mnt", "name": dc_name}],
-        "command": ["sh"],
-        "args": [
+    if cmd_args is None:
+        args = [
```

Comment:
This variable should be `cmd_args` as `args` is not used in `container_data`

Reply:
Done